bootstrap/terraform-fully-private/scripts/push-images-to-ecr.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
+
+AWS_REGION='us-east-1'
+AWS_ACCOUNT_ID=''
+
+# Choose to use docker or podman. Syntax is the same.
+#PROGRAM=podman
+PROGRAM=docker 
+
+${PROGRAM} pull docker.io/openpolicyagent/gatekeeper-crds:v3.16.3
```

Comment:
Have you tried using ECR cache pull trough images for these? If cache pull trough works, it should be easy to set these in the terraform code instead of in a disjoined file.